### ISSUE TITLE
[BUG][NOWEB] - send me.pushName field in WORKING "session.status" webhook event after first scan QR

### ISSUE THREAD

**Issue by AliSot2000** [allburov/whatsapp-http-api#409]
**Describe the bug**

The returned Response once the Session is in the State `WORKING` doesn't contain the `me.pushName` field despite it being required.

**Version**

```json
{
  "version": "2024.6.8",
  "engine": "NOWEB",
  "tier": "PLUS",
  "browser": "/usr/bin/chromium"
}
```

**To Reproduce** Steps to reproduce the behavior:

1. Start the Docker container AND have a webserver ready to receive the webhooks for `session.status`
2. Create a Session
3. Scan the QR Code
4. Observe the `me.pushName` Field is missing in the event for the session reaching the state `WORKING`

Example JSON I received:
```json
{
    "event": "session.status", 
    "session": "default", 
    "me": {
        "id": "11111111111@c.us"
    }, 
    "payload": {
        "name": "default", 
        "status": "WORKING"
    }, 
    "engine": "NOWEB", 
    "environment": {
        "version": "2024.6.8", 
        "engine": "NOWEB", 
        "tier": "PLUS", 
        "browser": "/usr/bin/chromium"
    }
}
```

**Expected behavior**
The me.pushName is populated also for the `WORKING` Event.

**Comment by devlikepro** [comment]
It happens when you scan qr code for the first time, after that the webhook will have all fields.
We'll find the right solution for that, thank you for such detailed issue!

Consider getting session details with `GET /api/sessions/{name}/` after you got the first WOKRING event for now

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
fixed in `2024.11.5`

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
